deploy/operator/kustomization.yaml

```diff
@@ -5,12 +5,3 @@ resources:
 - role.yaml
 - serviceaccount.yaml
 
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-images:
-- name: monitoring-stack-operator
-  newName: quay.io/sthaha/monitoring-stack-operator
-  newTag: 0.0.1
-
-commonLabels:
```

Comment:
commonLabels also sets deployment selector which are immutable and should not be changed.